src/frontend/src/content/docs/ja/whats-new/aspire-13-1.mdx

```diff
@@ -0,0 +1,591 @@
+---
+title: Aspire 13.1 の新機能
+description: Aspire 13.1 delivers comprehensive MCP support for AI coding agents, CLI enhancements with channel persistence, dashboard improvements, Azure Managed Redis, container registry support, and numerous bug fixes.
```

Comment:
I'm curious, why don't you translate the `description` too? I can't think of a reason not to.